examples/nanoChatGPT/train_rlhf.py

```diff
@@ -34,24 +36,24 @@ def __init__(self, critic):
             self.in_keys = critic.in_keys
             self.out_keys = critic.out_keys
             self.module = critic
-            
+
         def forward(self, tensordict):
             ndim = tensordict.ndim
             training = self.module.training
             self.module.eval()
-            td = vmap(self.module, (ndim-1,))(tensordict)
+            td = vmap(self.module, (ndim - 1,))(tensordict)
             self.module.train(training)
             # vmap sends this dim to the beginning so we need to send it back where it belongs
             td = td.permute(*range(1, ndim), 0)
             return tensordict.update(td)
-    
+
     vmap_critic = VmapCritic(critic)
 
     adv_fn = GAE(value_network=vmap_critic, gamma=0.99, lmbda=0.95, average_gae=True)
-    
+
     # FIXME: why not using the scheduler?
     # Loss
-    loss_fn = ClipPPOLoss(actor, critic, gamma=0.99)
+    loss_fn = ClipPPOLoss(actor, critic_head)
```

Comment:
is it right we removed gamma?

Reply:
yes it's only used to calculate the advantage which we do in the advantage module
If you're using the latest torchrl you should get a warning that tells you not to use gamma in the constructor